install.sh

```diff
@@ -215,19 +197,9 @@ function extract_install () {
     mkdir "${CREW_DEST_DIR}"
     cd "${CREW_DEST_DIR}"
 
-    #extract and install
+    # Extract and install
     echo_intra "Extracting ${1} ..."
-    if [[ "$2" == *".zst" ]];then
-      LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} tar -Izstd -xpf ../"${2}"
-    elif [[ "$2" == *".tpxz" ]];then
-      if ! LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} pixz -h &> /dev/null; then
-        tar xpf ../"${2}"
-      else
-        LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} tar -Ipixz -xpf ../"${2}"
-      fi
-    else
-      tar xpf ../"${2}"
-    fi
+    tar xpf ../"${2}"
```

Comment:
I agree we shouldn't be changing `LD_LIBRARY_PATH`.

Reply:
We don't need to when we're just using the system tar and such.